erpnext/buying/doctype/buying_settings/buying_settings.json

```diff
@@ -159,27 +159,27 @@
    "fieldtype": "Check",
    "label": "Set Landed Cost Based on Purchase Invoice Rate"
   },
-  {
-   "default": "0",
-   "description": "Percentage you are allowed to order more against the Blanket Order Quantity. For example: If you have a Blanket Order of Quantity 100 units. and your Allowance is 10% then you are allowed to order 110 units.",
-   "fieldname": "over_order_allowance",
-   "fieldtype": "Float",
-   "label": "Over Order Allowance (%)"
-  },
   {
    "default": "0",
    "description": "While making Purchase Invoice from Purchase Order, use Exchange Rate on Invoice's transaction date rather than inheriting it from Purchase Order. Only applies for Purchase Invoice.",
    "fieldname": "use_transaction_date_exchange_rate",
    "fieldtype": "Check",
    "label": "Use Transaction Date Exchange Rate"
+  },
+  {
+   "default": "0",
+   "description": "Percentage you are allowed to buy beyond the Blanket Order Quantity. For example: If you have a Blanket Order for a quantity of 100 units, and your allowance is 10%, then you are allowed to buy up to 110 units.",
```

Comment:
Do we really need the excessively verbose example?

Also I’d suggest lowercase quantity. “Blanket Order” is the doctype. Quantity is just a rather generic field on it. Excessive uppercase naming communicates a “Take extra care, this is something special!” kind of feeling, nd there’s already too much of this.